app/views/rockles/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<ul>
+
+  <% @rockles.each do |rockle| %>
+
+<li><%= rockle.name %>:
```

Comment:
I would have to see the view in my browser, but this might be better done in a table, with <tr> instead of <li> and <td> instead of <span>